Product/Production/Services/AuditRepositoryCore/pom.xml

```diff
@@ -51,4 +51,20 @@
             <artifactId>junit</artifactId>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
```

Comment:
The AuditTransform abstract class tests are written in AuditRepositoryCore services jar. Since these are same across all services PD,DQ,DR,DS,AD,X12 and in order to avoid repetition of writing unit tests for same code an abstract AuditTransform testclass is created and the AuditRepositoryCore  test jar is added as dependency to PatientDiscovery Core services. Same way it can be used in other Core services and the service specific AuditTransformer uni tests can be written in their respective services core.